### PR TITLE
Update Ambassador Page for Better Directions

### PR DESCRIPTION
**Pull request by Chaboi45**
Adds more guidance to our Ambassadors. Check the pictures below for a before/after.
Before(Current View):
Screen Shot 2021-01-24 at 3 56 17 PM
After(After Changes. Tested in the local environment):
Screen Shot 2021-01-24 at 3 56 02 PM
This will help get our Ambassadors in the right direction 👍

### CODE REVIEW

File: app/templates/contribute/ambassador.jade [hunk 1]
```diff
@@ -30,7 +30,7 @@ block content
   
       h4(data-i18n="contribute.how_to_join") How to Join
       p
-        a(href="https://discourse.codecombat.com/", data-i18n="community.social_discource")
+        a(href="https://discourse.codecombat.com/c/ambassador/6", data-i18n="community.social_discource")
```
Comment (Chaboi45) [inline app/templates/contribute/ambassador.jade:33]: Thoughts on changing it to a list of the following? Includes the link to wiki, slack channel, and the ambassador category. However, I'm not sure what or how the i18n system works, so I'm reluctant to push these changes unless `data-i18n` tags are updated on all locale files in GitHub.
```jade
ul
        li
          a(href="https://github.com/codecombat/codecombat/wiki/Ambassador-Home", data-i18n="") Read the documentation.
        li
          a(href="https://coco-slack-invite.herokuapp.com/", data-i18n="") Find us in our public Slack channel.
        li
          a(href="https://discourse.codecombat.com/c/ambassador/6", data-i18n="") Help others in the Ambassador category.
```

Comment (Chaboi45) [reply app/templates/contribute/ambassador.jade:33]: @AndrewJakubowicz care to take a look at this one?

**Comment by Chaboi45** [comment]
Just finished the draft. Ready to pull 👍

**Comment by Chaboi45** [comment]
Tested locally and is working 👍 Feel free to accept PR.
Screen Shot 2021-01-24 at 3 56 02 PM

**Review by nwinter — APPROVED** [review]
Great addition, thanks @Chaboi45!

**Pull request merged**